gating/pre_merge_test/run

```diff
@@ -82,13 +97,24 @@ function _configure_rpco {
     ## Use the openstack-ansible binary
     export ANSIBLE_BINARY="openstack-ansible"
     ## Bootstrap RPC-O's ansible
-    bash scripts/bootstrap-ansible.sh
-    cp ${CLONE_DIR}/tests/rpco_vars/user_rpc_ceph_vars.yml /etc/openstack_deploy/user_rpc_ceph_vars.yml
-    bash scripts/bootstrap-aio.sh
+    if [ "${RE_JOB_SCENARIO}" = "rpco_newton" ]; then
+      bash scripts/bootstrap-ansible.sh
+      cp ${CLONE_DIR}/tests/rpco_vars/user_rpc_ceph_vars.yml /etc/openstack_deploy/user_rpc_ceph_vars.yml
+      bash scripts/bootstrap-aio.sh
+    fi
+    if [ "${RE_JOB_SCENARIO}" = "rpco_master" ] || [ "${RE_JOB_SCENARIO}" = "rpco_pike" ]; then
```

Comment:
Can we drop this conditional and just assume this is the RPCO method when NOT newton moving forward?